### ISSUE TITLE
Editing a field (not entire layer) based on user

### ISSUE THREAD

**Issue by virtusmankind** [3liz/lizmap-web-client#668]
Hello. 
I'd like to make a field editable (not the entire postgis layer) only for chosen users.  
![cattura](https://user-images.githubusercontent.com/20582012/30074297-5680ebfa-9271-11e7-9d98-d72ef788927d.JPG)

It's a layer used to report issues to the municipality office. People fill the form to report the issue. The last field is the "Report state" set by default "Da risolvere" (To solve). All the user allowed to edit the layer could change the "Stato segnalazione" value to "Analyzed" or "Solved". I'd like to change this, allowing only chosen users to change that field. 
Should I set something on PostgreSQL? How can I relate DB with Lizmap user allowed?

**Comment by rldhont** [comment]
To do so, you have to use a table in which is stored the possible values to **Stato segnalazione**. Add this table to your qgis project, use it in the relational edition widget, and publish it in WFS.
In this table you add a field **group**, set the value to **admin** (or the choosen user group) and specified in Lizmap plugin that this table is filtered by user.
Then used Lizmap 3.1.4 or 3.0.16.

When you finalize this can you update the documentation https://github.com/3liz/lizmap-documentation and specifically this part https://github.com/3liz/lizmap-documentation/blob/master/source/publish/advanced_lizmap_config.rst#filtered-layers---filtering-data-in-function-of-users ?

**Comment by virtusmankind** [comment]
I created a csv table "Stato Segnalazione", then I uploaded it on postgis. It's a table with 3 columns (id, stato, group) and 3 records I need ("Da risolvere", "Presa in carico", "Risolta"). Each record has list of group allowed ("admins,tecnico"). I set the widget for field "**stato**" as **Value relation** using the table as source.

![cattura](https://user-images.githubusercontent.com/20582012/30640572-bb457c8e-9e03-11e7-8a9f-744917c299d5.JPG)

Using your procedure, groups not allowed to change the values of "Stato segnalazione" (mandatory field) won't see any value when create/edit an item (even if I set a "Default value" in Widget edit) and they cannot add a new item without a value set. Besides, they also see "Group" field in editing popup, even if I set it "hidden" with "all" as only value. No problems for allowed groups: they can add/edit the "Stato segnalazione" field and save the item, and they don't see "Group" field.
What's wrong? I'm using Lizmap 3.1.1
